Sources/JWTKit/ECDSA/ECDSACurve.swift

```diff
@@ -10,35 +10,35 @@
 /// The use of ``ECDSACurve`` in cryptographic operations allows for easy specification and interchange of
 /// the elliptic curves based on security requirements and application needs.
 public struct ECDSACurve: LosslessStringConvertible, Sendable {
-    let kind: Kind
+    let backing: Backing
     
     public var description: String {
-        kind.rawValue
+        backing.rawValue
     }
 
-    package static let p256 = Self(curve: .p256)
-    package static let p384 = Self(curve: .p384)
-    package static let p521 = Self(curve: .p521)
-    package static let ed25519 = Self(curve: .ed25519)
-    package static let ed448 = Self(curve: .ed448)
+    package static let p256 = Self(backing: .p256)
+    package static let p384 = Self(backing: .p384)
+    package static let p521 = Self(backing: .p521)
+    package static let ed25519 = Self(backing: .ed25519)
+    package static let ed448 = Self(backing: .ed448)
```

Comment:
Why `package` rather than `public`? Are these not publicly used? If not, you don't need the separate backing enum, source compatibility only applies to `public` symbols.

Reply:
If they _are_ supposed to be public, don't forget to add the `Codable` methods